src/moonlink/src/table_handler.rs

```diff
@@ -368,6 +395,17 @@ impl TableHandlerState {
             error!(error = ?e, "failed to notify force snapshot, because receiver end has closed channel");
         }
     }
+
+    fn start_alter_table(&mut self, alter_table_request: AlterTableRequest) {
+        // Alter table will block any events, so table must be at a consistent view.
+        assert!(self.table_consistent_view_lsn.is_some());
+        assert!(self.special_table_state == SpecialTableState::Normal);
+        self.largest_force_snapshot_lsn = Some(self.table_consistent_view_lsn.unwrap());
```

Comment:
`largest_force_snapshot_lsn` is, as its name suggests, the largest requested LSN.
On pg_mooncake side, it always requests for last committed transaction's commit LSN, which could be larger than `table_consistent_view_lsn` which comes from cdc.